File: Season.java
import java.util.TreeSet;

public class Season extends Transfarable{
  public int year;
  public String season;

  public Season(int year, String season){
    this.year = year; this.season = season;
    this.transfers = new TreeSet<Transfer>();
  }

  public String toString(){
    return season;
  }
}

File: Season.py
import Transferable

class Season(Transferable.Transferable):
    def __init__(self, year, season):
        super().__init__()
        self.year = year
        self.season = season

    def __str__(self):
        return self.season

    def __lt__(self, other):
        return self.season < other.season
